File: android/trunk/src/com/usedparts/domain/OrdersResult.java
package com.usedparts.domain;

import com.fasterxml.jackson.annotation.JsonProperty;

import java.util.List;

public class OrdersResult {
    @JsonProperty("pages")
    public int pages;

    @JsonProperty("total")
    public int total;

    @JsonProperty("items")
    public List<Order> items;
}

File: wp/trunk/UsedParts.Domain/OrdersResult.cs
using System.Collections.Generic;
using Newtonsoft.Json;

namespace UsedParts.Domain
{
    public class OrdersResult
    {
        [JsonProperty(PropertyName = "pages")]
        public int Pages { get; set; }

        [JsonProperty(PropertyName = "total")]
        public int Total { get; set; }

        [JsonProperty(PropertyName = "items")]
        public IEnumerable<Order> Items { get; set; }
    }
}
